### ISSUE TITLE
missing metrics

### ISSUE THREAD

**Issue by volkcj** [SouthForkResearch/CHaMP_Metrics#31]
## Metrics Requiring Coding moved below this line

*None*

## Resolved Metrics moved below this line

*The following metrics were originally missing from the validation report. The cause of these was identified as missing XPaths. These XPaths were fixed on 8 May 2017.*

thalweg depth avg
slow water frequency
fast non turbulent frequency
fast turbulent frequency
bankfull width avg
bankfull width cv
bankfull width to depth ratio avg
bankfull width to depth ratio cv
bankfull width to max depth ratio avg
bankfull width to max depth ratio cv
wetted depth avg
wetted depth max
wetted width avg
wetted width cv
wetted width to depth ratio avg
wetted width to depth ratio cv
wetted width to max depth ratio avg
wetted width to max depth ratio cv
wetted side channel width
wetted side channel width cv
wetted side channel width to depth ratio avg
wetted side channel width depth ratio cv
wetted side channel width to max depth ratio avg
wetted side channel width to max depth ratio cv
residual pool depth

**Comment by philipbaileynar** [comment]
Residual Pool Depth is metric ID 3372.

See also issue #36 which is related. Essentially residual pool depth is needed for:

* Tier 1
* Tier 2
* Visit Level

(It is working for channel unit level)

**Comment by philipbaileynar** [comment]
Visit level residual pool depth is a repeat of the tier 1 residual pool depth for `SlowPool`. Therefore this metric was directly implemented as part of issue #36. 

The final part of this solution was to implement the corresponding XPath in the Workbench database for Metric ID 3372.

```
/TopoMetrics/Metrics/ChannelUnitsTier1/SlowPool/ResidualDepth
```
